File: C/EX19.c
/*Calcula velocidade final e classifica entre lento e rápido*/
#include<stdlib.h>
#include<stdio.h>

int main(int argc, char const *argv[])
{
    float a, vi, t, vf;

    printf("Informe:\n\nAceleracao(m/s^2): ");
    scanf("%f", &a); 

    printf("\nVelocidade inicial(m/s): ");
    scanf("%f", &vi);

    printf("\nTempo(s): ");
    scanf("%f", &t);

    vf = vi + (a*t);

    if (40 < vf && vf <= 60)
    {
        printf("Velocidade permitida.");
    }
    
    else if (60 < vf && vf <= 80)
    {
        printf("Velodidade de cruzeiro.");
    }

    else if (80 < vf && vf <=120)
    {
        printf("Veiculo rapido.");
    }

    else if (vf > 120)
    {
        printf("Veiculo muito rapido.");
    }
    
    else
    {
        printf("Veiculo muito lento");
    }
    
    return 0;
}

File: Python/EX19.py
#classificação de acordo com a velocidade

a = float(input("Aceleração: "))
vi = float(input("Velocidade inicial: "))
t = float(input("Tempo: "))

v = vi + (a*t)

if (40 < v) and (v <= 60):
    print("Velocidade permitida.")

elif (60 < v) and (v <= 80):
    print("Velocidade de cruzeiro.")

elif (80 < v) and (v <= 120):
    print("Veículo rápido.")

elif v > 120:
    print("Veículo muito rápido.")

else:
    print("Veículo muito lento.")